cudax/include/cuda/experimental/__async/sender/conditional.cuh

```diff
@@ -289,7 +288,11 @@ _CUDAX_TRIVIAL_API auto __cond_t::__closure<_Pred, _Then, _Else>::__mk_sender(_S
     {}, static_cast<__cond_t::__data<_Pred, _Then, _Else>&&>(__data_), static_cast<_Sndr&&>(__sndr)};
 }
 
-_CCCL_GLOBAL_CONSTANT __cond_t conditional{};
+template <class _Sndr, class _Pred, class _Then, class _Else>
+inline constexpr int structured_binding_size<__cond_t::__sndr_t<_Sndr, _Pred, _Then, _Else>> = 3;
```

Comment:
Mild preference to use `size_t` which is consistent with other traits that hold sizes